xija/gui_fit/app.py

```diff
@@ -243,6 +243,18 @@ def __init__(self, model, main_window):
         main_box.addWidget(QtWidgets.QLabel("Timestep: {:.1f} s".format(model.dt)))
         main_box.addWidget(QtWidgets.QLabel("Evolve Method: Core {}".format(model.evolve_method)))
         main_box.addWidget(QtWidgets.QLabel("Runge-Kutta Order: {}".format(4 if model.rk4 else 2)))
+        for msid in self.model.limits:
+            limits = self.model.limits[msid]
+            units = limits["unit"]
+            main_box.addWidget(QtWidgets.QLabel(f"{msid.upper()} limits:"))
+        for limit, val in limits.items():
+            if limit == "unit":
+                continue
+            limit_str = f"    {limit}: {val:.1f} {units}"
+            if units == "degF":
+                valC = (val - 32.0)*5.0/9.0
```

Comment:
BTW, you can consolidate the temperature conversions (in 2 places) using `cheta.units.F_to_C`. The `cheta.units` import is lightweight.

Reply:
Thanks! Done